__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data =require("../db/data/test-data/index")
+
+afterAll(() => {
+  return db.end();
+});
+beforeEach(() => {
+  return seed(data);
+});
```

Comment:
Good work remembering to implement these jest lifecycle methods. Including seeding each test with test data before the tests run, this is great.